govstack_test_harness_api/building_blocks/bb-digital-registries/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views.check_if_record_exists_in_registry import check_if_record_exists_in_registry
+
+
+urlpatterns = [
+    path('data/<str:registryname>/<str:versionnumber>/exists', check_if_record_exists_in_registry)
```

Comment:
can you also add name to the url? ([django doc](https://docs.djangoproject.com/en/4.2/topics/http/urls/#naming-url-patterns))